cmd/tfimport/main.go

```diff
@@ -136,13 +137,19 @@ func run() error {
 
 		// Important to handle this last.
 		default:
-			return fmt.Errorf("import resource %q: %v", ir.Change.Address, err)
+			errMsg := fmt.Sprintf("failed to import resource %q: %v\n%v", ir.Change.Address, err, string(output))
+			errors += fmt.Sprintf("%v\n", errMsg)
```

Comment:
Consider the following:

var errs []string

for _, cc := ... {
   default:
       errs = append(errs, fmt.Sprintf(...))
}

if len(errs) > 0 {
  return fmt.Errorf("failed to import %v resources:\n%v, len(errs), strings.Join(errs, "\n")
}

Reply:
Good idea. I'm not sure how I forgot that strings.Join is a thing.